conda/install.py

```diff
@@ -227,17 +227,20 @@ def exp_backoff_fn(fn, *args):
     """Mostly for retrying file operations that fail on Windows due to virus scanners"""
     if not on_win:
         return fn(*args)
-    import random
 
+    import random
     max_retries = 5
     for n in range(max_retries):
         try:
             result = fn(*args)
-        except Exception as e:
+        except EnvironmentError as e:
```

Comment:
`PermissionError` in one case, the other I couldn't get to repeat, but hopefully the same.